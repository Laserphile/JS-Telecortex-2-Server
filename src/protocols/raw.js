export function colours2rgb(colours) {
  return colours.map(({ r, g, b }) => {
    return [r, g, b];
  });
}
