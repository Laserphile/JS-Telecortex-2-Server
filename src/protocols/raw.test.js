import { colours2rgb } from './raw';

require('../testing/setupTests.js');

describe('colours2rgb', () => {
  it('works', () => {
    expect(colours2rgb([{ r: 255, g: 0, b: 0 }])).toMatchSnapshot();
  });
});
