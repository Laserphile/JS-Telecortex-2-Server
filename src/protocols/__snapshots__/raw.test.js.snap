// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`colours2rgb works 1`] = `
Array [
  Array [
    255,
    0,
    0,
  ],
]
`;
